pylabnet/utils/iq_upconversion/iq_calibration.py

```diff
@@ -192,7 +191,6 @@ def get_ampl_phase(self, if_freq, lo_freq):
 
 		return q_ret, phase_ret
 
-
 	def get_dc_offsets(self, if_freq, lo_freq):
 		if (not self.initialized):
 			raise Exception("No calibration loaded!")
```

Comment:
Ideally, the `IQOptimizer` would have a `self.log: LogClient` member variable, such that all manual exceptions and print statements can be replaced by `self.log.error(error_msg)` and `self.log.info` messages. That way, the optimization routine will nicely integrate into future GUIs or scripts. 

The `self.log.error(error_msg)` will automatically raise an exception, and has the added benefit that the error message will be logged and potentially stored in a log-file.